scripts/prepForBuild.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const bundle = 'auro-alert__bundled.js';
+const indexFile = './build/index.html';
+
+// File destination.txt will be created or overwritten by default.
+let copyFiles = async function() {
+  fs.copyFile(`./dist/${bundle}`, `./build/${bundle}`, (err) => {
+    if (err) throw err;
+    console.log(`${bundle} was copied to ./build dir`);
+  });
+
+  fs.copyFile(`./demo/css/style.css`, `./build/css/style.css`, (err) => {
```

Comment:
This is a bug in the generator https://github.com/AlaskaAirlines/WC-Generator/issues/160

Please update to point to the correct CSS file.

Reply:
@blackfalcon shouldn't we be copying the .map file over as well?

Reply:
no, the .map file will get generated as needed.